File: PooCC01CNetz/Semana03/Java/Ejercicios2/Libro.java
public class Libro {
    private String isbn;
    private String titulo;
    private String autor;

    public Libro(String isbn, String titulo, String autor) {
        this.isbn = isbn;
        this.titulo = titulo;
        this.autor = autor;
    }

    public Libro() {}

    public void setIsbn(String isbn) {
        this.isbn = isbn;
    }

    public String getIsbn() {
        return isbn;
    }

    public void setTitulo(String titulo) {
        this.titulo = titulo;
    }

    public String getTitulo() {
        return titulo;
    }

    public void setAutor(String autor) {
        this.autor = autor;
    }

    public String getAutor() {
        return autor;
    }

    public void imprimirAtributos() {
        System.out.println("ISBN    : " + isbn);
        System.out.println("Titulo  : " + titulo);
        System.out.println("Autor   : " + autor);
    }
}

File: PooCC01CNetz/Semana03/Python/Ejercicios2/Libro.py
class Libro:
    def __init__(self, isbn = "", titulo = "", autor = ""):
        self.isbn = isbn
        self.titulo = titulo
        self.autor = autor

    def setIsbn(self, isbn):
        self.isbn = isbn

    def getIsbn(self):
        return self.isbn
    
    def setTitulo(self, titulo):
        self.titulo = titulo

    def getTitulo(self):
        return self.titulo
    
    def setAutor(self, autor):
        self.autor = autor

    def getAutor(self):
        return self.autor
    
    def imprimirAtributos(self):
        print("ISBN    : " + self.isbn)
        print("Titulo  : " + self.titulo)
        print("Autor   : " + self.autor)